style.css

```diff
@@ -0,0 +1,356 @@
+*{
+    margin: 0;
+    padding: 0;
+  }
+  :root {
+    /*Colors & Shadows*/
+
+        /*kind of cyan*/
+            --cyan: #5996A5;
+            --cyan-dark: #2B788B;
+            --cyan-light: #C3DCE3;
+
+        /*kind of pink*/
+            --pink: #A15993;
+            --pink-dark: #945069;
+            --pink-light: #F2D4DC;
+
+        /*kind of grey*/
+            --grey-light: #F6F5F4;
+            --grey-medium: #E0E0E0;
+            --grey-icon: #BABABA;
+            --grey-dark-one:#757575;
+            --grey-dark-two: #585858;
+
+        /*single*/    
+        --balck: #000000;
+        --green: #639B6D;
+        --red: #A95151;
+        --yellow: #C4A24C;
+        --Orange: #CB5B43;
+
+        /*shadows*/
+        --shadows:0px 22.8981px 45.7963px -11.4491px rgba(52, 41, 39, 0.08);
+
+    /*Fonts*/
+
+        /*headings*/
+            --heading-one: 400 56px/56px 'Dela Gothic One';
+            --heading-two: 400  48px/48px 'Dela Gothic One';
+            --heading-three: 400 32px/32px 'Dela Gothic One';
+            --heading-four: 400 24px/24px 'Dela Gothic One';
+            --heading-five: 400 16px/100% 'Dela Gothic One';
+            --heading-six: 400 32px/100% 'Dela Gothic One';
+            --heading-seven: 400 20px/100% 'Dela Gothic One';
+            --heading-eight: 400 24px/100% 'Dela Gothic One';
+
+        /*text*/
+            --body-one: 600 18px/25px 'Montserrat';
+            --body-two: 600 16px/22px 'Montserrat';
+            --body-three: 600 14px/20px 'Montserrat';
+            --body-four: 600 12px/17px 'Montserrat';
+            --body-five: 600 14px/140% 'Montserrat';
+            --body-six: 600 12px/140% 'Montserrat';
+            --body-seven: 600 12px/100% 'Montserrat';
+            --body-eight: 600 10px/100% 'Montserrat';
+            --body-nine: 600 16px/100% 'Montserrat';
+        
+        /*button*/
+            --button-one: 700 18px/25px 'Montserrat';
+            --button-two: 700 16x/22px 'Montserrat';
+            --button-three: 700 12x/100% 'Montserrat';
+            --button-four: 700 14x/100% 'Montserrat';
+
+        /*links*/
+            --link-one: 16px/20px 'Montserrat';
+            --link-two: 14px/20px 'Montserrat';
+    /*Padding*/
```

Comment:
It is not the best way to define paddings variables. Try to define the scale order, e.g.:

spacing-xxs: 2px
spacing-xs: 4px
spacing-s: 8px
spacing-m: 16px
....

Than use those tokens separately
padding: spacing-s
padding-right: spacing-m
...
etc

Not every padding value should be a variable